test/forms/inputs/dropdown-checkbox-group.test.js

```diff
@@ -1,39 +1,64 @@
-import React from 'react'
-import { mount } from 'enzyme'
+import React, { useState } from 'react'
 import { DropdownCheckboxGroup } from '../../../src/'
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
 
-test('DropdownCheckboxGroup adds value to array when unselected option clicked', () => {
-  const onChange = jest.fn()
-  const TOGGLED_OPTION = 'TOGGLED_OPTION'
-  const props = {
+const WrappedDropdownCheckboxGroup = (props) => {
+  const [value, setValue] = useState(props.value || [])
+
+  const options = [
+    { key: 'First Option', value: '1' },
+    { key: 'Second Option', value: '2' },
+    { key: 'Third Option', value: '3' },
+  ]
+
+  const defaultProps = {
     input: {
       name: 'test',
-      value: [],
-      onChange,
+      value: value,
+      onChange: (e) => setValue(e),
     },
     meta: {},
-    options: [TOGGLED_OPTION],
+    options,
   }
-  const wrapper = mount(<DropdownCheckboxGroup {...props} />)
-  wrapper.find('input').simulate('change')
-  const newValue = onChange.mock.calls[0][0]
-  expect(newValue).toEqual([TOGGLED_OPTION])
+
+  return <DropdownCheckboxGroup {...defaultProps} {...props} />
+}
+
+test('DropdownCheckboxGroup adds value to array when unselected option clicked', async () => {
+  render(<WrappedDropdownCheckboxGroup />)
+
+  const user = userEvent.setup()
+  const select = screen.getAllByRole('group')[0]
+
+  await user.click(select)
+
+  const firstCheckbox = screen.getByLabelText('First Option')
+  await user.click(firstCheckbox)
+
+  const thirdCheckbox = screen.getByLabelText('Third Option')
+  await user.click(thirdCheckbox)
+
+  expect(firstCheckbox).toBeChecked()
+  expect(thirdCheckbox).toBeChecked()
+
+  const selectValueLabel = screen.getByText('3, 1')
+  expect(selectValueLabel).toBeInTheDocument()
 })
 
-test('DropdownCheckboxGroup removes value to array when selected option clicked', () => {
-  const onChange = jest.fn()
-  const TOGGLED_OPTION = 'TOGGLED_OPTION'
-  const props = {
-    input: {
-      name: 'test',
-      value: [TOGGLED_OPTION],
-      onChange,
-    },
-    meta: {},
-    options: [TOGGLED_OPTION],
-  }
-  const wrapper = mount(<DropdownCheckboxGroup {...props} />)
-  wrapper.find('input').simulate('change')
-  const newValue = onChange.mock.calls[0][0]
-  expect(newValue).toEqual([])
+test('DropdownCheckboxGroup removes value from array when selected option clicked', async () => {
+  render(<WrappedDropdownCheckboxGroup value={['1']} />)
+
+  const select = screen.getAllByRole('group')[0]
+  const user = userEvent.setup()
```

Comment:
Let's move this above `render` (same goes for the rest of the tests in this file)